packages/core/src/dashboards/component/RefreshChart/RefreshChart.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useEffect, useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+import {ChartRender, Icon} from '@axelor/aos-mobile-ui';
+import {fetchChartById} from '../../api.helpers';
+import AOPChart from '../AOPChart/AOPChart';
+
+interface RefreshChartProps {
+  style?: any;
+  chartId: number;
+  hideCardBackground?: boolean;
+}
+
+const RefreshChart = ({
+  style,
+  chartId,
+  hideCardBackground = false,
+}: RefreshChartProps) => {
+  const [charConfig, setChartConfig] = useState({
+    dataList: [],
+    chartName: null,
+    chartType: null,
+    actionViewName: null,
```

Comment:
Create a const above component for the default value to avoid redefining it everywhere : 
```
const DEFAULT_CONFIG = {
    dataList: [],
    chartName: null,
    chartType: null,
    actionViewName: null,
};
```